Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:12.22.8-alpine as builder
+
+WORKDIR /app
+
+COPY . .
+
+RUN yarn install && yarn build
```

Comment:
Split yarn install and build into two statements
first only copy package.json and yarn.lock and install
this can speed up and use docker caching.